README.md

```diff
@@ -1 +1,144 @@
-# DELETE ME
+# ACM SIGDOC Structured Authoring article
+
+Articles that ACM SIGDOC Stuructured Authoring committee members write are hosted on this repository. Reviews take place by using the branching system, which separate articles into different branches during their review lifecycle. The final version of the articles and other resources are published on our committee resources website:
+
+- [ACM SIGDOC Structured Authoring Committee Resources](https://acm-sigdoc-structured.org/index.html)
+
+# How to contribute
+
+Before you contribute, reach out to us at authors@acm-sigdoc-structured.org. After onboarding, you can pick from a list of topics and begin writing your article.
+
+# General workflow
+
+We support articles written in Markdown or Microsoft Word. Regardless of what format you choose, the workflow process is broken down into five steps that are represented by branches respectively:
+
+- `initial_draft`
+- `technical_review`
+- `instructional_review`
+- `editorial_review`
+- `final_draft`
+
+Each branch is structured by the author, meaning you must create a folder with your name in each branch, if it doesn't exist yet. You can add additional subfolder for organization, but you don't have to.
+
+You must create your own new working branch at every step and work on your draft in that branch. Your working branch is always a copy of the latest review step branch that your draft was accepted into.
+
+To progress to the next step in the review process, create a pull request using your working branch and open it against one of the five review step branches that represents the review step that you are ready for.
+
+Your working branch can always be a copy of the latest review step branch that your article was merged into.
+
+For example, if you do not have a draft yet, you can create a branch named `myarticle1`. In that branch, write your article or upload it as a `.docx` file. Once you are ready for you draft to appear in the `initial_draft` branch, create a pull request with `myarticle1` as your source and `initial_draft` as your destination.
+
+The review occurs in the pull request. The reviewer can add comments, make suggestions, and directly approve the pull request.
+
+After your pull request is approved and merged, your draft appears in `initial_draft` and you can delete the `myarticle1` branch.
+
+Now, you can proceed to the next step by creating a copy of `initial_draft` named `myarticle2` and making any changes you need to it. When you are ready for technical review, follow the same steps as previously.
+
+After approval, you can merge the pull request and your tech reviewed article will appear in the `technical_review` branch. Repeat the process the remaining review steps.
```

Comment:
This section is long and was kind of a way for me to keep track of the process while writing it. I think we can either cut this down or remove it entirely, but it might also be helpful to have some of the basics here.